privacyidea/api/lib/utils.py

```diff
@@ -207,7 +208,7 @@ def send_csv_result(obj, data_key="tokens",
     # Do the data
     for row in obj.get(data_key, {}):
         for val in row.values():
-            if type(val) in [str, unicode]:
+            if isinstance(val, six.text_type):
```

Comment:
Hm, should we change this to ``six.string_types`` for Python 2 compatibility?

Reply:
Yes, missed that.

Reply:
fixed in 21bdef5b

Reply:
Thanks!